lib/mcb/courses_editor.rb

```diff
@@ -11,6 +11,8 @@ class CoursesEditor
       accredited_body: :accrediting_provider,
       start_date: :start_date,
       application_opening_date: :applications_open_from,
+      age_range: :age_range,
+      course_code: :course_code,
```

Comment:
was there a reason to add `course_code` only for 

` unless attribute_name == :course_code`

Reply:
Yes - when editing course code, the table library doesn’t like having two columns with the same name.

Reply:
I've changed that part of the code now so it's no longer in the hash: b0709d8

Reply:
Yeah i saw, the the more I ask the more magical it gets

Reply:
Then it’s easy - just ask fewer questions before approving :trollface: